### PR TITLE
Add TLS to Synergy (Lite edition)

### PR DESCRIPTION
**Pull request by SerhiiGadzhilov**
SYNERGY1-1583

### CODE REVIEW

File: src/gui/src/AppConfig.cpp [hunk 1]
```diff
@@ -440,7 +440,11 @@ bool AppConfig::isCryptoAvailable() const {
     bool result {true};
 
 #ifndef SYNERGY_ENTERPRISE
-    result = (edition() == kPro || edition() == kPro_China || edition() == kBusiness);
+    result = (edition() == kPro ||
+              edition() == kPro_China ||
+              edition() == kBusiness ||
+              edition() == kBasic_Ligth ||
```
Comment (nbolton) [inline src/gui/src/AppConfig.cpp:446]: Use instead: `kLite` (instead of `kBasic_Ligth`)

File: src/lib/shared/EditionType.h [hunk 2]
```diff
@@ -27,7 +27,9 @@ enum Edition {
     kUnregistered,
     kBusiness,
     kBasic_China,
-    kPro_China
+    kPro_China,
+    kBasic_Ligth,
```
Comment (nbolton) [inline src/lib/shared/EditionType.h:31]: Use instead: `kLite` (instead of `kBasic_Ligth`)

**Review by nbolton — CHANGES_REQUESTED** [review]

File: src/gui/src/AppConfig.cpp (hunk 1 above)
Comment (SerhiiGadzhilov) [reply src/gui/src/AppConfig.cpp]: Fixed

File: src/lib/shared/SerialKeyEdition.h [hunk 3]
```diff
@@ -43,6 +43,8 @@ class SerialKeyEdition
 	static const std::string BASIC_CHINA;
 	static const std::string BUSINESS;
 	static const std::string UNREGISTERED;
+	static const std::string ULTIMATE;
+	static const std::string BASIC_LIGHT;
```
[the remark this answers is not in the document]
Comment (SerhiiGadzhilov) [reply src/lib/shared/SerialKeyEdition.h]: Fixed

**Review by nbolton — APPROVED** [review]

**Pull request merged**